quint/src/names/base.ts

```diff
@@ -20,21 +21,37 @@ import { QuintType } from '../ir/quintTypes'
 export type DefinitionKind = 'module' | 'def' | 'val' | 'assumption' | 'param' | 'var' | 'const' | 'type'
 
 /**
- * A minimal representation of a QuintDef, to be stored in `LookupTable` and
- * `DefinitionsByName`.
+ * A definition to be stored in `DefinitionsByName` and `LookupTable`. Either a `QuintDef`
+ * or a `QuintLambdaParameter`, with additional metadata fields
+ *
+ * A definition can be hidden, meaning it
```

Comment:
There's something missing here :)

Reply:
I meant to delete these field details from the type definition since I added individual comments for each field (a few lines below). Forgot to do that after I changed my mind from typing stuff here to typing the individual comments. I'll just delete it now.